KubeArmor/monitor/systemMonitor.go

```diff
@@ -329,6 +329,21 @@ func (mon *SystemMonitor) UpdateThrottlingConfig() {
 		cfg.GlobalCfg.ThrottleSec)
 }
 
+// UpdateUntrackedNamespaces updates the runtime untracked namespace list.
+
+func (sm *SystemMonitor) UpdateUntrackedNamespaces(v string) {
+	parts := strings.Split(v, ",")
+
+	namespaces := make([]string, 0, len(parts))
+	for _, ns := range parts {
+		if ns = strings.TrimSpace(ns); ns != "" {
+			namespaces = append(namespaces, ns)
+		}
+	}
+
```

Comment:
need to update `cfg.Globalcfg.untrackedns` as well
```suggestion
cfg.Globalcfg.untrackedns = namespaces
```